models/bamf_pet_ct_breast_tumor/config/default.yml

```diff
@@ -0,0 +1,49 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet Breast FDG-avid lesions segmentation (dicom to dicom)
+
+execute:
+- FileStructureImporter
+- SitkNiftiConverter
```

Comment:
Why do we need this converter? Cannot we use the [NiftiConverter](https://github.com/MHubAI/documentation/blob/main/documentation/mhubio/mhubio_modules.md#nifticonverter) module? There you can configure the backend to use dcm2niix which offers high performance.